tools/cbmc/proofs/Queue/QueueGenericCreate/Configurations.json

```diff
@@ -0,0 +1,68 @@
+{
+  "ENTRY": "QueueGenericCreate",
+  "CBMCFLAGS":
+  [
+      "--unwind 1",
+      "--signed-overflow-check",
+      "--pointer-overflow-check",
+      "--unsigned-overflow-check"
```

Comment:
no memory safety flags?

Reply:
The code is written in a way, that on boot the device uses pvPortMalloc to allocate a couple of data structures for reuse which are never freed while the device is only. Therefore the memory safety flags is not delivering very meaningful results as these data structures are supposed to be not freed.